From 73fb869fa115da82fe598fef0103ff43cc5ef38c Mon Sep 17 00:00:00 2001
From: Elaine Zhang <zhangqing@rock-chips.com>
Date: Fri, 9 Sep 2016 09:17:22 +0800
Subject: [PATCH] ARM64: dts: rockchip: rk3399: add edp power domain

add pd_edp node.
add the edp needed power domain node for rk3399.

Change-Id: Ie1a4a7013b0d5cfc2b0180f701341a6b977a637a
Signed-off-by: Elaine Zhang <zhangqing@rock-chips.com>
---
 arch/arm64/boot/dts/rockchip/rk3399-android.dtsi | 1 +
 arch/arm64/boot/dts/rockchip/rk3399.dtsi         | 4 ++++
 2 files changed, 5 insertions(+)

diff --git a/arch/arm64/boot/dts/rockchip/rk3399-android.dtsi b/arch/arm64/boot/dts/rockchip/rk3399-android.dtsi
index 0e963032c667..28e4d5de1ccf 100644
--- a/arch/arm64/boot/dts/rockchip/rk3399-android.dtsi
+++ b/arch/arm64/boot/dts/rockchip/rk3399-android.dtsi
@@ -502,6 +502,7 @@
 		resets = <&cru SRST_P_EDP_CTRL>;
 		reset-names = "edp_apb";
 		status = "disabled";
+		power-domains = <&power RK3399_PD_EDP>;
 	};
 };
 
diff --git a/arch/arm64/boot/dts/rockchip/rk3399.dtsi b/arch/arm64/boot/dts/rockchip/rk3399.dtsi
index 5411ac6fdf2c..5319b88363fe 100644
--- a/arch/arm64/boot/dts/rockchip/rk3399.dtsi
+++ b/arch/arm64/boot/dts/rockchip/rk3399.dtsi
@@ -993,6 +993,10 @@
 			};
 
 			/* These power domains are grouped by VD_LOGIC */
+			pd_edp@RK3399_PD_EDP {
+				reg = <RK3399_PD_EDP>;
+				clocks = <&cru PCLK_EDP_CTRL>;
+			};
 			pd_emmc@RK3399_PD_EMMC {
 				reg = <RK3399_PD_EMMC>;
 				clocks = <&cru ACLK_EMMC>;
-- 
2.35.3

